report_gen/README.md

```diff
@@ -0,0 +1,157 @@
+
+#### 2. Plot on multiple custom x axes
+
+The input x axes do not have to match with the columns perfectly; e.g. -x y would plot with x axes y1,y2,y3.
+
+##### Example:
+	python graphgen.py -f example/a.csv example/b.csv example/c.csv -o example/figs/graphgen -x z y2
+
+##### Generated graphs: 
+
+|example/figs/graphgen/all:y2-x0.svg|example/figs/graphgen/all:y2-y1.svg|example/figs/graphgen/all:y2-y3.svg|example/figs/graphgen/all:y2-z4.svg|
+:-------------------------:|:-------------------------:|:-------------------------:|:-------------------------:
+<img src="./example/figs/graphgen/all:y2-x0.svg" width="300" height="auto"> | <img src="./example/figs/graphgen/all:y2-y1.svg" width="300" height="auto"> | <img src="./example/figs/graphgen/all:y2-y3.svg" width="300" height="auto">|<img src="./example/figs/graphgen/all:y2-z4.svg" width="300" height="auto">
+
+|example/figs/graphgen/all:z4-x0.svg|example/figs/graphgen/all:z4-y1.svg|example/figs/graphgen/all:z4-y2.svg|example/figs/graphgen/all:z4-y3.svg|
+:-------------------------:|:-------------------------:|:-------------------------:|:-------------------------:
+<img src="./example/figs/graphgen/all:z4-x0.svg" width="300" height="auto"> | <img src="./example/figs/graphgen/all:z4-y1.svg" width="300" height="auto"> | <img src="./example/figs/graphgen/all:z4-y2.svg" width="300" height="auto">|<img src="./example/figs/graphgen/all:z4-y3.svg" width="300" height="auto">
+
+#### 3. Only plot specific columns
+
+The input column names do not have to match with the columns perfectly; e.g. --plot_col y would plot columns y1,y2,y3.
+
+##### Example:
+	python graphgen.py -f example/a.csv example/b.csv example/c.csv -o example/figs/graphgen --plot_col y1 z
+
+##### Generated graphs: 
+|example/figs/graphgen/all:x0-y1.svg|example/figs/graphgen/all:x0-z4.svg|
+:-------------------------:|:-------------------------:
+<img src="./example/figs/graphgen/all:x0-y1.svg" width="300" height="auto"> | <img src="./example/figs/graphgen/all:x0-z4.svg" width="300" height="auto"> 
+
+
+#### 4. Set extension of output graphs
+
+    Supported extensions are: "png", "jpeg", "webp", "svg", "pdf", "eps" and "html". 
+
+    "html" creates an interactive plotly graph.
+
+##### Example:
+    python graphgen.py -f example/a.csv example/b.csv example/c.csv -o example/figs/graphgen --plot_col y1 -e png svg html
+
+##### Generated graphs: 
+|example/figs/graphgen/all:x0-y1.png|example/figs/graphgen/all:x0-y1.svg|
+:-------------------------:|:-------------------------:
+<img src="./example/figs/graphgen/all:x0-y1.png" width="300" height="auto"> | <img src="./example/figs/graphgen/all:x0-y1.svg" width="300" height="auto">
+
+**example/figs/graphgen/all:x0-y1.html**
+<script src=\"https://cdn.plot.ly/plotly-latest.min.js\"></script><iframe frameborder=0 width="600" height="400" src=./example/figs/graphgen/all:x0-y1.html></iframe>
```

Comment:
This  <script> line is ugly and does not work on github, just write see example in example/... or create screenshot instead